visualization/app/codeCharta/ui/viewCube/viewCube.component.ts

```diff
@@ -35,22 +40,24 @@ export class ViewCubeController implements CameraChangeSubscriber, ViewCubeEvent
 	}
 
 	constructor(
-		private $element,
-		private $rootScope: IRootScopeService,
-		private threeOrbitControlsService: ThreeOrbitControlsService,
-		private viewCubeMouseEventsService: ViewCubeMouseEventsService
-	) {
-		"ngInject"
+		@Inject(ElementRef) private elementReference: ElementRef,
+		@Inject(ThreeOrbitControlsServiceToken) private threeOrbitControlsService: ThreeOrbitControlsService,
+		@Inject(ViewCubeMouseEventsService) private viewCubeMouseEventsService: ViewCubeMouseEventsService
```

Comment:
Also rename it to `viewCubeMouseEvents` here? Or keep `viewCubeMouseEventsService` in codeMap.mouseEvent.service

Reply:
changed to viewCubeMouseEvents  in next push.